lldb/source/Plugins/Process/gdb-remote/ProcessGDBRemote.cpp

```diff
@@ -2809,7 +2809,22 @@ void ProcessGDBRemote::RefreshStateAfterStop() {
   UpdateThreadListIfNeeded();
 
   if (m_last_stop_packet)
-    SetThreadStopInfo(*m_last_stop_packet);
+    std::string packet_str = m_last_stop_packet->GetStringRef().str();
```

Comment:
I think a better way might be to make sure we only handle GPUActions once per stop-id. I believe the new synchronization stuff Greg added might help keep track of this.

@clayborg What are your thoughts on the best way to fix this?